spec/controllers/posts_controller_spec.rb

```diff
@@ -190,6 +190,7 @@
       it "updates post with expected attributes" do
         new_title = RandomData.random_sentence
         new_body = RandomData.random_paragraph
+        my_post.user = my_user
```

Comment:
This line, is a fix for the spec. It associates the post to the user, so that the `authorize_user` filter lets the user update the post (cf. line 105 above).